lib/Runner.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
-const AWS = require('aws-sdk');
 const EventEmitter = require('events');
+const https = require('https');
```

Comment:
Why two groups?

Reply:
I usually separate 

- built ins
- third party
- locals

do you want to just combine builtin and third party?